react/components/QuoteDetails/QuoteDetails.tsx

```diff
@@ -112,6 +117,21 @@ const QuoteDetails: FunctionComponent = () => {
   /**
    * GraphQL Queries
    */
+  const userEmail = sessionResponse?.namespaces?.profile?.email?.value
+  const { data: userData } = useQuery(CHECK_PERMISSIONS, {
+    variables: { email: userEmail },
+  })
+
+  const costCenterId: string = userData?.getUserByEmail?.[0]?.costId
+  const { data: costCenterData } = useQuery(GET_COSTCENTER_BY_ID, {
+    variables: { id: costCenterId },
+  })
+
+  const orgId: string = userData?.getUserByEmail?.[0]?.orgId
+  const { data: organizationData } = useQuery(GET_ORGANIZATION_BY_ID, {
+    variables: { id: orgId },
```

Comment:
One of the advantages of graphql is that you don't need to do separate fetches to get information. Since you need all three of these to send the metric, you can build a single graphql query joining them all together in a single request. Also, it's probably better to build a new query for this use case anyway, since you probably don't need the exact same data that those existing queries were asking for. You can drop the unused fields and join the queries together, resulting in a better API request.